src/ARCExpect/ValidationFunctions.fs

```diff
@@ -165,6 +165,7 @@ module Validate =
                     if predicate en.Current |> not then
                         ErrorMessage.ofIParam $"does not satisfy predicate" en.Current
                         |> Expecto.Tests.failtestNoStackf "%s"
+                        loop ()
```

Comment:
Have you tested wether this continues after 1 `failtestNoStackf` is executed? If that throws an exception i am quite sure that it wont

Reply:
You are correct. It is more complicated to do this than I thought...
But I think I'll get a solution.